src/components/Footer/Footer.js

```diff
@@ -75,99 +64,23 @@ const Footer = ({
-        const selectedUser = {
-          user: {
-            name: channelName,
-            image: item?.actor?.data?.profile_pic_url || DEFAULT_PROFILE_PIC_PATH
-          }
-        };
-        const members = [item?.actor?.id, profile.myProfile.user_id];
-        await createSignChat(members, selectedUser);
-      } else {
-        await sendMessageDM(item.id, 'post', 'SIGNED');
-      }
-    } catch (e) {
-      console.warn(e);
-    } finally {
-      refSheet.current.close();
-      setLoading({...loading, loadingDm: false});
-    }
-  };
-
-  const onPressDMAnon = async () => {
-    try {
-      setLoading({...loading, loadingDmAnon: true});
-      await sendMessageDM(item.id, 'post', 'ANONYMOUS');
-    } catch (e) {
-      console.warn(e);
-    } finally {
-      refSheet.current.close();
-      setLoading({...loading, loadingDmAnon: false});
-    }
-  };
-
-  const dataSheet = [
-    {
-      id: 1,
-      name: loading.loadingDm ? 'Loading...' : `Message ${username}`,
-      icon: <SendDMBlack />,
-      onPress: onPressDM
-    },
-    {
-      id: 2,
-      name: loading.loadingDmAnon ? 'Loading...' : `Message ${username} anonymously`,
-      icon: <SendDMAnonBlock />,
-      onPress: onPressDMAnon
-    },
-    {
-      id: 3,
-      name: 'Share link',
-      icon: <ShareBlack />,
-      onPress: onPressShare
-    }
-  ];
-
   return (
     <View style={[styles.rowSpaceBeetwen, styles.container]}>
       <View style={styles.leftGroupContainer}>
-        {isShowDM && !isSelf ? (
-          <TouchableOpacity
-            testID="sendDM"
-            style={styles.btn}
-            onPress={() => refSheet.current.open()}>
-            <View style={styles.btnDM}>
-              <View style={styles.card}>
-                <MemoIc_senddm height={20} width={21} />
-                <Text style={styles.textDM}>DM</Text>
-              </View>
-            </View>
-          </TouchableOpacity>
-        ) : (
-          <TouchableOpacity testID="shareBtn" style={styles.btn} onPress={onPressShare}>
-            <View style={styles.btnShare}>
-              <MemoIc_share height={20} width={21} />
-            </View>
-          </TouchableOpacity>
-        )}
+        <TouchableOpacity testID="shareBtn" style={styles.btn} onPress={onPressShare}>
+          <View style={styles.btnShare}>
+            <MemoIc_share height={20} width={21} />
+          </View>
+        </TouchableOpacity>
         {disableComment ? (
           <View testID="disableComment" style={styles.btn}>
-            <View style={styles.btnComment(isShowDM, isSelf)}>
+            <View style={styles.btnComment}>
               <MemoIc_comment height={24} width={25} />
             </View>
           </View>
         ) : (
           <TouchableOpacity testID="availableComment" style={styles.btn} onPress={onPressComment}>
-            <View style={styles.btnComment(isShowDM, isSelf)}>
+            <View style={styles.btnComment}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for sending direct messages (DMs) has been removed, and the UI elements associated with it have also been removed. This includes the `onPressDM` and `onPressDMAnon` functions, the `dataSheet` array, and the `BottomSheetMenu` component. If this feature is no longer needed, this change is fine. However, if it's still required, you'll need to reintroduce this code.
```diff
-  const username = item?.anon_user_info_emoji_name
-    ? `Anonymous ${item?.anon_user_info_emoji_name}`
-    : item?.actor?.data?.username;
-
-  const onPressDM = async () => {
-    try {
-      setLoading({...loading, loadingDm: true});
-      if (!item?.anon_user_info_emoji_name) {
-        const channelName = [username, profile?.myProfile?.username].join(',');
-        const selectedUser = {
-          user: {
-            name: channelName,
-            image: item?.actor?.data?.profile_pic_url || DEFAULT_PROFILE_PIC_PATH
-          }
-        };
-        const members = [item?.actor?.id, profile.myProfile.user_id];
-        await createSignChat(members, selectedUser);
-      } else {
-        await sendMessageDM(item.id, 'post', 'SIGNED');
-      }
-    } catch (e) {
-      console.warn(e);
-    } finally {
-      refSheet.current.close();
-      setLoading({...loading, loadingDm: false});
-    }
-  };
-
-  const onPressDMAnon = async () => {
-    try {
-      setLoading({...loading, loadingDmAnon: true});
-      await sendMessageDM(item.id, 'post', 'ANONYMOUS');
-    } catch (e) {
-      console.warn(e);
-    } finally {
-      refSheet.current.close();
-      setLoading({...loading, loadingDmAnon: false});
-    }
-  };
-
-  const dataSheet = [
-    {
-      id: 1,
-      name: loading.loadingDm ? 'Loading...' : `Message ${username}`,
-      icon: <SendDMBlack />,
-      onPress: onPressDM
-    },
-    {
-      id: 2,
-      name: loading.loadingDmAnon ? 'Loading...' : `Message ${username} anonymously`,
-      icon: <SendDMAnonBlock />,
-      onPress: onPressDMAnon
-    },
-    {
-      id: 3,
-      name: 'Share link',
-      icon: <ShareBlack />,
-      onPress: onPressShare
-    }
-  ];
-
   return (
     <View style={[styles.rowSpaceBeetwen, styles.container]}>
       <View style={styles.leftGroupContainer}>
        {isShowDM && !isSelf ? (
          <TouchableOpacity
            testID="sendDM"
            style={styles.btn}
            onPress={() => refSheet.current.open()}>
            <View style={styles.btnDM}>
              <View style={styles.card}>
                <MemoIc_senddm height={20} width={21} />
                <Text style={styles.textDM}>DM</Text>
              </View>
            </View>
          </TouchableOpacity>
        ) : (
          <TouchableOpacity testID="shareBtn" style={styles.btn} onPress={onPressShare}>
            <View style={styles.btnShare}>
              <MemoIc_share height={20} width={21} />
            </View>
          </TouchableOpacity>
        )}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->